src/rpc-request.js

```diff
@@ -42,9 +41,18 @@ var rpcRequest = function (path, body, accessToken, selectUser) {
       body = null;
     }
 
-    apiRequest = request.post(BASE_URL + path)
-      .type('application/json')
-      .set('Authorization', 'Bearer ' + accessToken);
+    apiRequest = request.post(getBaseURL(host) + path)
+      .type('application/json');
+
+    switch (auth) {
+      case 'user':
```

Comment:
I had missed a change in `generate_routes.py`; that's why the team routes weren't changed. I fixed that and fixed up one resulting test failure.

Now I realize there's not a lot of testing around the team endpoints. :-) (Notably, nothing tests that they have an auth header.) That makes me a little nervous about merging this, but I also don't really want to spend a lot of time testing. Is there someone else who can add some tests or at least manually verify that the basics are working?